src/mozillavpn.cpp

```diff
@@ -1238,11 +1238,13 @@ void MozillaVPN::update() {
 
   setUpdating(true);
 
+#ifndef MVPN_WINDOWS
```

Comment:
Can you write 1 line of comment to explain why we have this #ifndef ?

Reply:
Will do